web/packages/design/src/Button/Button.tsx

```diff
@@ -158,23 +158,22 @@ const buttonStyle = <E extends React.ElementType>(
   const { fill, intent } = props;
   const palette = buttonPalette(props);
   return {
-    backgroundColor: palette.default.background,
+    backgroundColor: palette.default.background ?? 'transparent',
```

Comment:
After making `background` required in `ButtonPaletteEntry`, there are four places where the background is missing. That's because in most places we provide `TextAndBackgroundColors` which already requires `background` to be present. What do you think about explicitly typing them out instead of having an implicit default here?

This won't quite work for `border`, as there are many more type errors after making `border` required.